lib/uplink/bucket.go

```diff
@@ -122,6 +122,36 @@ func (b *Bucket) UploadObject(ctx context.Context, path storj.Path, data io.Read
 	return errs.Combine(err, upload.Close())
 }
 
+// StreamObject streams a new object, if authorized.
+func (b *Bucket) StreamObject(ctx context.Context, path storj.Path, opts *UploadOptions) (w io.Writer, err error) {
```

Comment:
This adds a new method to the libuplink API. @jtolds @thepaul what do you think about it?

It seems that the FUSE integration takes advantage of something that is not available in the metainfo API, but is not available in the libuplink API.

Reply:
removed